site/content/docs/automate/automation-framework.md

```diff
@@ -25,9 +30,25 @@ If you are using the framework in the ZAP stable [docker](/docs/docker/about/) i
 ```bash
 docker run -v $(pwd):/zap/wrk/:rw -t owasp/zap2docker-stable bash -c "zap.sh -cmd -addonupdate; zap.sh -cmd -autorun /zap/wrk/zap.yaml"
 ```
-The weekly and live docker images are less likely to be out of date but if you want to use an add-on which has been updated since they were released then you should use the same sort of command.
 
-A longer term solution is being worked on.
+#### Weekly, Live and Post 2.11
```

Comment:
```suggestion
#### Weekly, Live, and Post 2.11
```

Reply:
Doh! one day I'll remember the Oxford comma!